views/peeps/index.erb

```diff
@@ -0,0 +1,8 @@
+<h1>"Welcome to Chitter!"</h1>
+<form action='/sign_up' method='get'>
+  <button name='Sign up' type='submit'>Sign up</button>
```

Comment:
Some are <button> elements and some are <input> forms. Fine if it's for styling, otherwise good to be consistent.